tests/functional/_data/files/.magento.env.yaml.scdondemand

```diff
@@ -0,0 +1,3 @@
+stage:
```

Comment:
Leave original extension of files to have it correctly parsed in IDE. Let's rename to `scdondemand.magento/env.yaml`

Reply:
Sure.